mutacc_auto/parse/parse_scout.py

```diff
@@ -39,3 +40,98 @@ def get_cases_from_scout(scout_output, days_ago=None):
             recent_cases.append(case)
 
     return recent_cases
+
+
+
+
+
+def get_vcf_from_json(scout_vcf_output):
+
+    """
+        Reconstructs vcf from scout variant object
+
+        Args:
+            scout_vcf_output (str): string returned by command 'scout export variants --json'
+
+        Returns:
+            vcf_string (str): string with vcf content
+    """
+
+    scout_vcf_output = json.loads(scout_vcf_output)
+
+    vcf_string = ""
+
+    #Write header of vcf
+    for header_line in HEADER:
+        vcf_string += header_line + '\n'
+
+    #Get samples
+    samples = [sample['sample_id'] for sample in scout_vcf_output[0]['samples']]
+    samples = '\t'.join(samples)
+
+    vcf_string += f"#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\t{samples}\n"
+
+
+    #Write variants
+    for variant in scout_vcf_output:
+
+        entry= []
+        entry.append(str(variant['chromosome'])) #CHROM
```

Comment:
Why do you not make this into a for loop and append each iteration using the key as iterator.
something like:
entry = []
for column in scout_header_column_names:
if column in variant:
  entry.append(str(variant[column] or '.'))   # Since it was only uninitilized place

Reply:
Handle the PASS outside loop if it is not part of variant

Reply:
Make this into a def